dev/benchmarks/macrobenchmarks/lib/src/web/bench_paths.dart

```diff
@@ -0,0 +1,6057 @@
+Path path63;
+Path path66;
+Path path69;
+Path path72;
+Path path75;
+Path path78;
+Path path80;
+Path path82;
+Path path84;
+Path path86;
+Path path88;
+Path path119;
+Path path120;
+Path path121;
+Path path122;
+Path path123;
+Path path125;
+Path path127;
+Path path129;
+Path path131;
+Path path132;
+Path path134;
+Path path137;
+Path path140;
+Path path143;
+Path path145;
+Path path147;
+Path path208;
+Path path209;
+Path path210;
+Path path211;
+Path path213;
+Path path216;
+Path path219;
+Path path222;
+Path path225;
+Path path227;
+Path path229;
+Path path232;
+Path path235;
+Path path238;
+Path path240;
+Path path242;
+Path path277;
+Path path278;
+Path path279;
+Path path280;
+Path path281;
+Path path282;
+Path path284;
+Path path286;
+Path path288;
+Path path290;
+Path path292;
+Path path295;
+Path path298;
+Path path301;
+Path path330;
+Path path331;
+Path path332;
+Path path333;
+Path path334;
+Path path336;
+Path path338;
+Path path340;
+Path path342;
+Path path344;
+Path path345;
+Path path346;
+Path path349;
+Path path352;
+Path path356;
+Path path358;
+Path path359;
+Path path360;
+Path path361;
+Path path362;
+Path path363;
+Path path366;
+Path path369;
+Path path372;
+Path path373;
+Path path374;
+Path path375;
+Path path376;
+Path path379;
+Path path382;
+Path path385;
+Path path386;
+Path path387;
+Path path388;
+Path path389;
+Path path392;
+Path path395;
+Path path398;
+Path path399;
+Path path400;
+Path path401;
+Path path402;
+Path path405;
+Path path408;
+Path path411;
+Path path412;
+Path path413;
+Path path414;
+Path path415;
+Path path418;
+Path path421;
+Path path424;
+Path path425;
+Path path426;
+Path path427;
+Path path428;
+Path path431;
+Path path434;
+Path path437;
+Path path438;
+Path path439;
+Path path440;
+Path path441;
+Path path444;
+Path path447;
+Path path450;
+Path path451;
+Path path452;
+Path path453;
+Path path454;
+Path path457;
+Path path460;
+Path path463;
+Path path464;
+Path path465;
+Path path466;
+Path path467;
+Path path470;
+Path path473;
+Path path476;
+Path path477;
+Path path478;
+Path path479;
+Path path480;
+Path path483;
+Path path486;
+Path path489;
+Path path490;
+Path path491;
+Path path492;
+Path path493;
+Path path496;
+Path path499;
+Path path502;
+Path path503;
+Path path504;
+Path path505;
+Path path506;
+Path path509;
+Path path512;
+Path path515;
+Path path516;
+Path path517;
+Path path518;
+Path path519;
+Path path522;
+Path path525;
+Path path528;
+Path path529;
+Path path530;
+Path path531;
+Path path532;
+Path path535;
+Path path538;
+Path path541;
+Path path542;
+Path path543;
+Path path544;
+Path path545;
+Path path548;
+Path path551;
+Path path554;
+Path path555;
+Path path556;
+Path path557;
+Path path558;
+Path path561;
+Path path564;
+Path path573;
+Path path577;
+Path path579;
+Path path591;
+Path path592;
+Path path593;
+Path path594;
+Path path595;
+Path path597;
+Path path599;
+Path path601;
+Path path603;
+Path path606;
+Path path608;
+
+void createPaths() {
```

Comment:
What is the source of these path ops? How do we update them later?

Reply:
Recording of each SurfacePath call and hooks into runPath. Not feasible to update. Unless all material components get reimplemented with new paint functions, the order/count will likely not change.

Reply:
Let's leave a comment explaining that this is a recording of the Flutter Gallery on May 2020. Otherwise, one year from now it will be very unclear what this is.

Reply:
Done.